src/transformers/models/bert_generation/modeling_bert_generation.py

```diff
@@ -570,7 +570,7 @@ def prepare_inputs_for_generation(self, input_ids, past=None, attention_mask=Non
         if past is not None:
             input_ids = input_ids[:, -1:]
 
-        return {"input_ids": input_ids, "attention_mask": attention_mask}
+        return {"input_ids": input_ids, "attention_mask": attention_mask, "past_key_values": past}
```

Comment:
@patil-suraj we forgot to add this in the BERT cache PR. Bert-like models can also be used as stand-alone `BertForCausalLM` models -> so we need to return `past_key_values` here.